### DISCUSSION TITLE
Full Path

### DISCUSSION THREAD

**Discussion by Crothas** [plausible/analytics#4854] [Self-Hosted Support]
Hello I have a site that I wrote in php and it uses args like omnidex_id and player to view data for the card game Grand Archive
example 
https://silvie.gg/event_decklists?omnidex_id=14889

But I am only getting https://silvie.gg/event_decklists in my plausible dashboard. I found PL_TRACK_FULL_URLS=true and have put it in my script and in the env override file for docker. But I don't seem to be getting the full urls still. Is this feature not available on the CE?

**Comment by ruslandoga** [comment]
👋 @Crothas 

I don't think `PL_TRACK_FULL_URLS` is a thing: https://github.com/search?q=repo%3Aplausible%2Fanalytics%20PL_TRACK_FULL_URLS&type=code

And AFAIK query strings are always dropped for privacy reasons. If you look at the code, there are no conditionals or configuration to keep them. UTM tags and referrers are extracted from query strings, but no other information is stored from them.
